packages/components/tabs/src/Tabs.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import type { CommonProps } from '@contentful/f36-core';
 import { TabsContextProvider } from './tabsContext';
+
+const calculateIndexMap = (children) => {
+  const tabListArray = React.Children.toArray(children).find(
```

Comment:
I'm not really sure about the solution. I would question why do we need to have an initially focused item at all.

The tabs can be rendered anywhere on the page and some other elements can be focused by default.

Reply:
I'd say that nothing should be really focused on when the component is rendered.